core/domain/draft_upgrade_services_test.py

```diff
@@ -106,6 +110,26 @@ def test_try_upgrade_success(self):
                 self.DRAFT_CHANGELIST, 1, exploration.version, self.EXP_ID),
             self.DRAFT_CHANGELIST)
 
+    def test_try_upgrade_from_v32_to_v33(self):
+        exp_id = 'exp1'
+        self.save_new_exp_with_states_schema_v0(
+            exp_id, self.USER_ID, 'Old Title')
+        exploration_model = exp_models.ExplorationModel.get(
+            exp_id, strict=True, version=None)
+        for i in python_utils.RANGE(31):
+            exploration_model.commit(
+                self.USER_ID, 'Changed title %s.' % i, [])
+        exploration_model.commit(
+            self.USER_ID, 'Migrate from v32 to v33', [{
+                'cmd': exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION,
+                'from_version': 32,
+                'to_version': 33
+                }])
```

Comment:
Deindent by 4 to match the indentation of the line where the opening bracket is.